src/azure/cli/commands/__init__.py

```diff
@@ -142,8 +142,9 @@ def __init__(self, name, handler, description=None, table_transformer=None):
         self.table_transformer = table_transformer
 
     def add_argument(self, param_name, *option_strings, **kwargs):
+        dest = kwargs.pop('dest', None)
         argument = CliCommandArgument(
-            param_name, options_list=option_strings, **kwargs)
+            dest or param_name, options_list=option_strings, **kwargs)
```

Comment:
Not familiar with general command parsing, so just confirm this is a needed change.